eosio_build.sh

```diff
@@ -207,6 +207,11 @@
 		MONGOD_CONF=/usr/local/etc/mongod.conf
 		OPENSSL_ROOT_DIR=/usr/local/opt/openssl
 	fi
+   
+   ${PWD}/scripts/clean_old_install.sh
+   if [ $? -ne 0 ]; then
+      exit -1
```

Comment:
Need to print an error message, in case clean_old_install.sh doesn't during its own execution.